core/geometry/src/Geometry.ts

```diff
@@ -255,6 +255,13 @@ export class Geometry {
     }
     return distance;
   }
+  /** Correct `fraction` to zero if undefined or smaller than fraction tolerance. Otherwise return it unchanged. */
```

Comment:
I'm not happy with this doc. I prefer

```
/**
 * Correct `fraction` to `replacement` if fraction is `undefined` or smaller than `Geometry.smallFraction`. Otherwise return it unchanged. 
 * If no `replacement` is passed, use 0.
```